mon-pix/tests/acceptance/challenge-item_test.js

```diff
@@ -188,6 +189,15 @@ describe('Acceptance | Displaying a challenge of any type', () => {
               // given
               assessment = server.create('assessment', 'ofCertificationType');
               server.create('challenge', 'forCertification', data.challengeType, 'withFocused');
+              class FocusedCertificationChallengeWarningManagerStub extends Service {
+                hasConfirmedFocusChallengeScreenWarning() {
+                  return true;
+                }
+              }
```

Comment:
C'est un stub, alors qu'on est en test d'acceptance :thinking: 
On pourrait 
- utiliser la vraie implémentation
- clear dans un after dans ce test ou en global (voir `mon-pix/tests/test-helper.js`)

```suggestion
              class FocusedCertificationChallengeWarningManagerStub extends Service {
                hasConfirmedFocusChallengeScreenWarning() {
                  return true;
                }
              }
```